config/locales/en.yml

```diff
@@ -1327,6 +1327,9 @@ in_person_proofing.headings.state_id_milestone_2: Enter the information on your
 in_person_proofing.headings.switch_back: Switch back to your computer to prepare to verify your identity in person
 in_person_proofing.headings.update_address: Update your current address
 in_person_proofing.headings.update_state_id: Update the information on your ID
+in_person_proofing.post_office_closed.body: Post Office locations will resume regular hours on Friday, January 10, 2025.
```

Comment:
@eileen-nava I added a sentence to the body copy late on Friday -- could you please update this so the body is:

> **You will not be able to visit a Post Office on January 9 to finish verifying your identity.** Post Office locations will resume regular hours on Friday, January 10, 2025.